expiry/plugin.php

```diff
@@ -955,7 +955,7 @@ function expiry_stats_response( $infos ) {
 yourls_add_filter( 'add_new_link', function ( $return, $url , $keyword, $title ) { 
 
 	// this method tolelrates no error in short url creation
-	if(isset ( $return['code'] ) ) {
+	if( isset ( $return['status'] ) &&  $return['status'] === 'fail' ) {
```

Comment:
or `isset ( $return['code'] ) && $return['code'] !== ''`, both conditions are valid.

Reply:
Both solutions work for me.